js/metroInit.js

```diff
@@ -33,62 +33,77 @@ UIFreewallVertical = (function() {
 
   UIFreewallVertical.setupFreewalVertical = function(data) {
     var wall = new freewall('[data-metro-id="' + data.id + '"]:not([data-mce-bogus] [data-metro-id="' + data.id + '"])');
-    var cellWidth;
-    var cellHeight;
+
+    function resizePanels() {
+      var containerWidth = $('[data-metro-id="' + data.id + '"]:not([data-mce-bogus] [data-metro-id="' + data.id + '"])').parent().width();
+      var $panels = $('.panels.size11, .panels.size21, .panels.size22');
```

Comment:
Why specify all the size classes when you need to find all the panels anyway?

```suggestion
      var $panels = $container.find('.panels');
```

Reply:
Fixed